go/teams/handler.go

```diff
@@ -386,11 +386,12 @@ func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase
 		// verify the invite info:
 
 		// find the invite in the team chain
-		invite, found := team.chain().FindActiveInviteByID(untrustedInviteeFromGregor.InviteID)
+		inviteMD, found := team.chain().FindActiveInviteMDByID(untrustedInviteeFromGregor.InviteID)
 		if !found {
 			g.Log.CDebugf(ctx, "FindActiveInviteByID failed for invite %s", untrustedInviteeFromGregor.InviteID)
 			return libkb.NotFoundError{Msg: "Invite not found"}
 		}
+		invite := inviteMD.Invite
```

Comment:
I wonder if we should ensure here that this is not a new-style invite

Reply:
Don't we switch on the types just below? Not sure what for.

Reply:
(We can of course if you think it's safer)